src/control-plane/backend/lambda/api/service/quicksight/sql-builder.ts

```diff
@@ -13,12 +13,13 @@
 
 import { format } from 'sql-formatter';
 import { ExploreComputeMethod, ExploreConversionIntervalType, ExploreGroupColumn, ExplorePathNodeType, ExplorePathSessionDef, ExploreRelativeTimeUnit, ExploreTimeScopeType, MetadataPlatform, MetadataValueType } from '../../common/explore-types';
+import { logger } from '../../common/powertools';
 
 export interface Condition {
   readonly category: 'user' | 'event' | 'device' | 'geo' | 'app_info' | 'traffic_source' | 'other';
   readonly property: string;
   readonly operator: string;
-  readonly value: string;
+  readonly value: any[];
```

Comment:
add test cases for the value has more than one item

Reply:
covered by "condition sql" test case